File: Latihan_a/Disk.cpp
#include "Processor.cpp"

class Disk {

    private:
        // atribut
        string type;
        string capacity;
        int price;

    public:
        // konstruktor
        Disk(){}

        Disk(string type, string capacity, int price)
        {
            this->type = type;
            this->capacity = capacity;
            this->price = price;
        }

        // setter
        void setType(string type)
        {
            this->type = type;
        }
        
        void setCapacity(string capacity)
        {
            this->capacity = capacity;
        }

        void setPrice(int price)
        {
            this->price = price;
        }

        // getter
        string getType()
        {
            return this->type;
        }
        
        string getCapacity()
        {
            return this->capacity;
        }

        int getPrice()
        {
            return this->price;
        }

        // destruktor
        ~Disk(){}
};

File: Latihan_b/Php/Disk.php
<?php

class Disk {

    // atribut
    private $type;
    private $capacity;
    private $price;

    // konstruktor

    function __construct($type = "",  $capacity = "", $price = 0)
    {
        $this->type = $type;
        $this->capacity = $capacity;
        $this->price = $price;
    }

    // setter
    function setType($type)
    {
        $this->type = $type;
    }
        
    function setCapacity( $capacity)
    {
        $this->capacity = $capacity;
    }

    function setPrice($price)
    {
        $this->price = $price;
    }

    // getter
    function getType()
    {
        return $this->type;
    }
        
    function getCapacity()
    {
        return $this->capacity;
    }

    function getPrice()
    {
        return $this->price;
    }
}

?>